hls4ml/converters/keras_to_hls.py

```diff
@@ -232,7 +232,10 @@ def keras_to_hls(config):
             if model_arch is None:
                 raise ValueError('No model found in config file.')
             else:
-                model_arch = json.loads(model_arch.decode('utf-8'))
+                # model_arch should be a string by default. Keeping this if just for compatibility.
```

Comment:
I tested this, and it seems to be an issue dating back to h5py version 3.0.0. Can you update this comment to include that? It will be easier in the future when we decide to clean up the codebase